### ISSUE TITLE
Remove ncollide3d usage

### ISSUE THREAD

**Issue by repi** [EmbarkStudios/physx-rs#17]
Currently the `physx` crate uses `ncollide3d` for a couple of things, we should try and remove that usage as it feels unnecessary to bring in a different collision system in addition to physx.

**Comment by repi** [comment]
It is only this function that uses it in `heightfield.rs` so should be pretty easy to remove.

```Rust
    /// Build the heightfield into a PhysX geometry and a trimesh we can use for rendering.
    pub fn build_with_trimesh(&self, cooking: &mut Cooking) -> (Geometry, TriMesh<f32>) {
        let heights = self.generate_heights();
        let heightfield_desc = self.create_desc(&heights);

        let vertices = self.generate_vertices(&heights);
        (
            cooking.create_heightfield(heightfield_desc, true),
            quad_with_vertices(&vertices[..], self.size.0, self.size.1),
        )
    }
```

**Comment by MaikKlein** [comment]
I talked to @tgolsson and this function is not used atm. Either we remove it or we triangulate it ourselves.

I'll nuke ncollide/nalgebra from the repo now.

**Comment by repi** [comment]
Sweet! 

#22 is the issue of converting `nalgebra` to `glam`

**Comment by repi** [comment]
Fixed in #25
